notebooks/features/vw/Vowpal Wabbit - Classification using SparkML Vector.ipynb

```diff
@@ -0,0 +1,208 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img width=\"200\" src=\"https://mmlspark.blob.core.windows.net/graphics/emails/vw-blue-dark-orange.svg\" />\n",
+    "\n",
+    "# Binary Classification with VowalWabbit on Criteo Dataset \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## SparkML Vector input"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Read dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pyspark.sql import SparkSession\n",
+    "\n",
+    "# Bootstrap Spark Session\n",
+    "spark = SparkSession.builder.getOrCreate()\n",
+    "\n",
+    "from synapse.ml.core.platform import *\n",
+    "\n",
+    "from synapse.ml.core.platform import materializing_display as display"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyspark.sql.types as T\n",
+    "from pyspark.sql import functions as F\n",
+    "\n",
+    "schema = T.StructType(\n",
+    "    [\n",
+    "        T.StructField(\"label\", T.IntegerType(), True),\n",
+    "        *[T.StructField(\"i\" + str(i), T.IntegerType(), True) for i in range(1, 13)],\n",
+    "        *[T.StructField(\"s\" + str(i), T.StringType(), True) for i in range(26)],\n",
+    "    ]\n",
+    ")\n",
+    "\n",
+    "df = (\n",
+    "    spark.read.format(\"csv\")\n",
+    "    .option(\"header\", False)\n",
+    "    .option(\"delimiter\", \"\\t\")\n",
+    "    .schema(schema)\n",
```

Comment:
perhaps save this so that schema can be inferred correctly

Reply:
I was about to change and convert to parquet, but this a snippet from a public dataset (criteo) and it's useful to know how to convert the data